examples/redis/cookbooks/custom-redis/attributes/default.rb

```diff
@@ -0,0 +1,97 @@
+default['redis'].tap do |redis|
+  # Set to true if you want to install from source
+  # Installing from the Gentoo package in the portage tree is faster,
```

Comment:
I updated the comment to say that installing from packages is the recommended approach.